src/main/java/com/pinterest/secor/util/CompressionUtil.java

```diff
@@ -37,9 +38,21 @@
     public static CompressionCodec createCompressionCodec(String className)
             throws Exception {
         Configuration configuration = new Configuration();
-        CompressionCodecFactory.setCodecClasses(configuration,new LinkedList<Class>(Collections.singletonList(Class.forName(className))));
+        CompressionCodecFactory.setCodecClasses(configuration,new LinkedList<>(Collections.singletonList(Class.forName(className))));
         CompressionCodecFactory ccf = new CompressionCodecFactory(
                 configuration);
         return ccf.getCodecByClassName(className);
     }
-}
+
+    /**
+     * Retrieve Avro compression codec from config in param : avro.compression.codec
+     * List of default supported avro compression code
+     * @see org.apache.avro.file.CodecFactory
+     *
+     * @param config secor configuration
+     * @return codec as a String or null (No compression if not set)
+     */
+    public static String getAvroCompressionCodec(SecorConfig config) {
+        return (config.getAvroCompressionCodec() != null && config.getAvroCompressionCodec().toLowerCase() == "none") ? null : config.getAvroCompressionCodec();
```

Comment:
This coding is a bit weird.  It will be easier to read if coded like this:

return config.getAvro() == null || config.getAvro.equalsIgnoreCase("none") ? null : config.getAvro();

Also how does the caller deal with with the null return?